Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+      description: The URL category folder.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.type
+      description: The URL category type.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.list
+      description: The URL category match list.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.description
+      description: The URL category description.
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
+      isArray: false
+      name: folder
+      predefined:
+      - Shared
+      - Mobile Users
+      - Remote Networks
+      - Service Connections
+      - Mobile Users Container
+      - Mobile Users Explicit Proxy
+      required: false
+    - auto: PREDEFINED
+      default: false
+      isArray: false
+      description: The custom URL category's type.
+      name: type
+      predefined:
+      - URL List
+      - Category Match
+      required: true
+    - default: false
+      description: If the type is url list the value will be a comma separated array of url addresses. If the type is Category Match the value will be a comma separated array of categories name. The user can get the names by running prisma-sase-url-access-profile-list.
+      isArray: true
+      name: value
+      required: true
+    - default: false
+      isArray: false
+      name: description
+      description: The custom URL category's description.
+      required: false
+    - default: false
+      isArray: false
+      name: name
+      required: true
+      description: The name of the custom URL category.
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    execution: false
+    description: Create a new url category.
+    name: prisma-sase-custom-url-category-create
+    outputs:
+    - contextPath: PrismaSase.CustomURLCategory.id
+      description: The URL category id.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.name
+      description: The URL category name.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.folder
+      description: The URL category folder.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.type
+      description: The URL category type.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.list
+      description: The URL category match list.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.description
+      description: The URL category description.
+      type: String
+  - arguments:
+    - default: false
+      isArray: false
+      name: id
+      required: true
+      description: The custom URL category id.
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
```

```suggestion
      description: Prisma SASE folder location.
```
Is this the folder location or folder type/name?